src/scenes/CreateEvent/store.js

```diff
@@ -132,9 +121,13 @@ export default class CreateEventStore {
   @observable outcomes = INIT.outcomes
   @observable resultSetter = INIT.resultSetter // address
   @observable error = INIT.error
+
   @computed get hasEnoughFee() {
-    const transactionFee = _.sumBy(this.txFees, ({ gasCost }) => Number(gasCost));
-    return (this.app.wallet.lastUsedWallet.qtum >= transactionFee) && (this.app.wallet.lastUsedWallet.bot >= this.escrowAmount);
+    const transactionFee = sumBy(this.txFees, ({ gasCost }) => Number(gasCost));
+    const { currentWalletAddress } = this.app.wallet;
+    return currentWalletAddress
+      && (currentWalletAddress.qtum >= transactionFee)
+      && (currentWalletAddress.bot >= this.escrowAmount);
   }
   @computed get warning() {
```

Comment:
Yup this will cause a problem. When you try to run the lang script for bringing the strings to the other translations it will not detect it. Will fix.

Reply:
Oh this is already handled partly in one of my other PRs. We should do separate PR entirely to catch all the missing defined messages.